src/app/CommandHandler.h

```diff
@@ -57,6 +57,32 @@
 namespace chip {
 namespace app {
 
+/// Defines an abstract class of something that can be encoded
+/// into a TLV with a given data tag
+class EncoderToTLV
```

Comment:
Why is this CommandHandler-specific?  And if it is, why is it in the chip::app namespace?

Reply:
Was the minimal delta. The correct thing is probably for this to live close to DataModel::Encode since it ends up being like that. I will move it as a followup.